### ISSUE TITLE
PatchTST results

### ISSUE THREAD

**Issue by liuxu77** [thuml/Time-Series-Library#130]
Hi, thanks for supporting PatchTST in this library. I ran the scripts of PatchTST provided in this repository and noticed that the performance of PatchTST over the datasets of ETTh1 and ETTh2 seems to have a gap to the performance reported in the original paper. I wonder if this is due to the implementation difference. Thanks.

The reproduced results:
ETTh1 96 MSE 0.381  MAE 0.398
ETTh1 192 MSE 0.422  MAE 0.426
ETTh1 336 MSE 0.463  MAE 0.449
ETTh1 720 MSE 0.531  MAE 0.502

ETTh2 96 MSE 0.291  MAE 0.339
ETTh2 192 MSE 0.383  MAE 0.403
ETTh2 336 MSE 0.404  MAE 0.423
ETTh2 720 MSE 0.425  MAE 0.443

The performance gap becomes significant in the long-term scenarios, e.g., 336 and 720.

**Comment by wuhaixu2016** [comment]
Hi, the repo is aiming to provide a fair comparison to all the forecasting models.

Thus, we unify all the input length as 96 and set the training epochs as 10. In contrast, PatchTST adopts longer input length (that is 336) and set the training epochs as 100 with carefully tuned learning rate.

If you want to reproduce their paper, you may find a lot detailed configurations in their official code: https://github.com/yuqinie98/PatchTST

**Comment by liuxu77** [comment]
Hi, may I know why the test batch size is set to 1 for long-term forecasting tasks in this repo? I noticed a result difference when setting batch size to 1 and 32. Could you provide some insights on why this happen? Thanks.
